routes/confirm.js

```diff
@@ -46,26 +22,23 @@ router.post('/', function(req, res, next){
 		}
 	}
 
-    var crust = data.crusts[req.body[crust]];
+    var crust = data.crusts[req.body['crust']];
         
     // Add size cost
-    var size= data.sizes[req.body[size]];
+    var size= data.sizes[req.body['size']];
         
-    var topping = 0;
+    var toppingValue = 0;
```

Comment:
the var topping sharred the same name as our loop variable.

this was causing a mess